File: bindings/js/src/parsers/OpenAPIParser.ts
/**
 * Created by antoniogarrote on 05/05/2017.
 */

import {BaseParser} from "./BaseParser";
import {Clojure} from "../Clojure";

/**
 * Wrapper class for the AMF OpenAPI parser, processes OpenAPI specification documents and generate the DocumentModel out of them
 */
export class OpenAPIParser extends BaseParser {
    protected parser(): any {
        return new Clojure.amf.__GT_OpenAPIParser();
    }
}

File: bindings/java/src/org/raml/amf/parsers/OpenAPIParser.java
package org.raml.amf.parsers;

/**
 * Created by antoniogarrote on 04/05/2017.
 */

/**
 * Wrapper class for the AMF OpenAPI parser, processes OpenAPI specification documents and generate the DocumentModel out of them
 */
public class OpenAPIParser extends BaseParser {
    @Override
    protected String parserConstructor() {
        return "->OpenAPIParser";
    }
}
